docs/build/quick-start.md

```diff
@@ -1,303 +1,138 @@
-
-=== "Output after anchor"
-
-    ```bash
-    {
-      "type": 0,
-      "content": {
-        "Foo": "Bar"
-      },
-      "metadata": {
-        "unique": "E4qPslUd0qo98TZX",
-        "schema": null,
-        "controllers": [
-          "did:key:z6MkfZ6S4NVVTEuts8o5xFzRMR8eC6Y1bngoBQNnXiCvhH8H"
-        ]
-      },
-      "signature": 2,
-      "anchorStatus": "ANCHORED",
-      "log": [
-        {
-          "cid": "bagcqceranhr345kxf5gb3kwjvvnnefn2krqvhkpnurphqrn3lj773mfszmza",
-          "type": 0
-        },
-        {
-          "cid": "bafyreig6hostufw42cmz2cnn7hpvb6pau67a2n2syhzej7orqxfymdayyq",
-          "type": 2
-        }
-      ],
-      "anchorScheduledFor": null,
-      "anchorProof": {
-        "root": "bagcqceranhr345kxf5gb3kwjvvnnefn2krqvhkpnurphqrn3lj773mfszmza",
-        "txHash": "bagjqcgza4xgkpjodtqtgyu2fx6rdr6fb6mhevd5hy4253tl6pjlssidpwaha",
-        "chainId": "eip155:3",
-        "blockNumber": 9527752,
-        "blockTimestamp": 1611485094
-      }
-    }
+    await doc.update(newContent, newMeta, opts)
     ```
+<!-- API Reference here -->
 
-    !!! quote ""
-        This output was seen after the anchor has been created. The stream state has now shifted *anchorStatus* to `ANCHORED`. You can also see that the log contains one more entry.
 
-## **4. Update a stream**
-Use the `update` command to update a stream. Your [DID](../learn/glossary.md#dids) must be the [controller](../learn/glossary.md#controllers) of the stream in order to update it. Note that *TileDocument* is the only StreamType that can currently be updated by the CLI.
+In the following example we update a TileDocuments' content while also giving it a tag.
 
 === "Command"
 
-    ```bash
-    $ ceramic update kjzl6cwe1jw147ww5d8pswh1hjh686mut8v1br10dar8l9a3n1wf8z38l0bg8qa --content '{
-        "Foo": "Baz"
-      }'
-    ```
-    
-    !!! quote ""
-        You should use your StreamID instead of the StreamID included here.
-
-=== "Output"
+    ```JavaScript
+    const streamId = "kjzl6cwe1jw147ww5d8pswh1hjh686mut8v1br10dar8l9a3n1wf8z38l0bg8qa"
+    const doc = await TileDocument.load(ceramic, streamId)
 
-    ```bash
-    {
-      "Foo": "Baz"
-    }
+    await doc.update({foo: 'baz'}, {tags: ['baz']})
     ```
-
-??? info "More options"
-    Currently you can change *content*, *controllers*, and *schema* using the CLI. Run `ceramic update -h` for more information.
-
+<!-- API Reference here -->
 
 ## **5. Create a schema**
```

Comment:
I wonder if we should even include this schema stuff in the quick start, or just leave it in the TileDocument specific docs.  Doesn't seem as relevant as the basic CRUD ops for a quick start guide

Reply:
My argument for including it is just to streamline the flow to the TileDocument docs. It's never a bad thing to have more links to the docs in my eyes.

Reply:
Ignore this, I thought there was somewhere else in the docs that had an API Reference. I was wrong so this isn't going to stay.